.zenodo.json

```
@@ -0,0 +1,162 @@
+{
+    "creators": [
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Vay, Jean-Luc"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Almgren, Ann"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Amorim, L\u00edgia Diana"
+      },
+      {
+        "affiliation": "Laboratoire d'Optique Appliqu\u00e9e",
+        "name": "Andriyash, Igor"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Belkin, Daniel"
```

@danielbelkin what is your ORCiD? :)
```suggestion
        "name": "Belkin, Daniel",
       "orcid": "..."
```

---

https://orcid.org/0000-0002-3936-6419